### PR TITLE
Docs for GitLab

### PR DESCRIPTION
**Pull request by Fishbowler**
Docs for using GitLab.

* Assumes we've published v1.3 and have the `hosts` option.
* Also includes a tiny fix for the GitHub docs, where I stole the structure and all of the config detail from.

Shame there's no good GitLab-specific splash on Unsplash :shrug:

### CODE REVIEW

**Comment by Fishbowler** [comment]
wrt the hosts option, I've also opened https://gitlab.com/xmpp-interop-testing/xmpp-interop-testing-gitlab-component/-/merge_requests/5 to use it more properly in the repo

File: _documentation/gitlab.md [hunk 1]
```diff
@@ -1,11 +1,109 @@
...
 author: Dan Caseley
 ---
 
-Coming soon.
+## Introduction
 
-_Splash image courtesy of [Pavel Neznanov, Unsplash](https://unsplash.com/photos/silver-and-gold-round-accessory-w95Fb7EEcjE?utm_content=creditShareLink&utm_medium=referral&utm_source=unsplash)_
+This guide is intended for XMPP Server developers who are building in GitLab. It demonstrates how to add XMPP Interoperability and Standards Conformance tests integrated into a build & test pipeline. This assumes that you already have a pipeline established, and wish to add the additional checks.
+
+Like most external jobs in GitLab, this one consists of both an included component and an explicit to call it as a job. When called, it runs the tests, giving junit-esque outputs and debug logs. The only prerequisite is that you've got a built server, and have started it.
```
Comment (guusdk) [inline _documentation/gitlab.md:13]: This is a bit of a confusing sentence. Did you mean "Like most external jobs in GitLab, this one consists of both an included component and an explicit component to call it as a job."

**Review by guusdk — APPROVED** [review]
Thanks! This looks great. I have one minor question related to wording.

File: _documentation/gitlab.md (hunk 1 above)
Comment (Fishbowler) [reply _documentation/gitlab.md]: I think I improved it. How about now?

**Pull request merged**